src/index.html

```diff
@@ -11,7 +11,100 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="formBlock">
+        <legend>Personal information</legend>
+        <label class="classForm">Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="classForm">Name:
+          <input type="text" name="name">
+        </label>
+        <label class="classForm">How old are You?
+          <input
+            value="12"
+            type="number"
+            name="birth"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="classForm">Full date of birth:
```

Comment:
```suggestion
        </label>
        
        <label class="classForm">
          Full date of birth:
```
also add empty line between siblings